network/transparent_vlan_endpointclient_linux.go

```diff
@@ -194,9 +194,9 @@ func (client *TransparentVlanEndpointClient) setLinkNetNSAndConfirm(name string,
 	// confirm veth was moved successfully
 	err = RunWithRetries(func() error {
 		// retry checking in the namespace if the interface is not detected
-		return ExecuteInNS(client.nsClient, client.vnetNSName, func() error {
-			_, ifDetectedErr := client.netioshim.GetNetworkInterfaceByName(client.vlanIfName)
-			return errors.Wrap(ifDetectedErr, "failed to get vlan veth in namespace")
+		return ExecuteInNS(client.nsClient, nsName, func() error {
+			_, ifDetectedErr := client.netioshim.GetNetworkInterfaceByName(name)
```

Comment:
Does this mean it was checking the wrong interface? If yes, won't this fail consistently.
How was this function successful prior to this change?